packages/api/src/routes/events.ts

```diff
@@ -1,9 +1,16 @@
-import {Epoch, Number64, phase0, Slot, ssz, StringType, RootHex} from "@chainsafe/lodestar-types";
+import {Epoch, Number64, phase0, Slot, ssz, StringType, RootHex, altair} from "@chainsafe/lodestar-types";
 import {ContainerType, Json, Type} from "@chainsafe/ssz";
 import {jsonOpts, RouteDef, TypeJson} from "../utils";
 
 // See /packages/api/src/routes/index.ts for reasoning and instructions to add new routes
 
+export type LightclientHeaderUpdate = {
+  syncAggregate: altair.SyncAggregate;
+  header: phase0.BeaconBlockHeader;
+  /** Precomputed root to prevent re-hashing */
+  blockRoot: RootHex;
```

Comment:
Won't the client have to re-hash the header anyway for verification?

Reply:
Yeah it's a left-over from some testing. I agree on removing it